src/UvUnwrap.cpp

```
@@ -20,9 +20,153 @@
+    std::vector<uint32_t> triChart(triCount, 0);
+    if (triCount == 0)
+        return triChart;
+
+    std::unordered_set<uint64_t> seamSet(seamEdges.begin(), seamEdges.end());
+
+    auto triIncluded = [&](size_t t) {
+        return includeTri.empty() || (t < includeTri.size() && includeTri[t]);
+    };
+
+    std::vector<std::vector<size_t>> triNeighbors(triCount);
+    for (size_t t0 = 0; t0 < triCount; ++t0) {
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_ENUM is a macro then please configure it.

(unknownMacro)